src/PersistStore.ts

```diff
@@ -24,7 +25,7 @@ import {
 
 export class PersistStore<T, P extends keyof T> {
   private cancelWatch: IReactionDisposer | null = null;
-  private properties: string[] = [];
+  private properties: SerializableProperty<T, P>[] = [];
```

Comment:
I think would should support the old way of just strings and not require that you need to create a `SerializableProperty` for each property.

```suggestion
  private properties: (string | SerializableProperty<T, P>)[] = [];
```

If you want you can take the string property that comes in and make it a SerializableProperty within the `mobx-persist-store` code so it is easier to work with.

Reply:
You need to have more attention for this code) 

You may pass both to the `properties` in the `storageOptions`, but: 
- if you pass string property it will be converted to the SerializableProperty 
- if you pass SerializableProperty it will be uses as-is

This way makes possible works only with one type (SerializableProperty) in the PersistStore.